server/inc/server.h

```diff
@@ -14,7 +14,7 @@ typedef struct s_client {
     GOutputStream *ostream;
     GDataInputStream *data_in;
     GDataOutputStream *data_out;
-    gint64 uid;
+    gint32 uid;
```

Comment:
It's better to remove this and add auth_token that also will need for another feature

Reply:
No, it would be in new variable in structure

Reply:
why is uid so important for u? IMHO, it's trash.